cnf-certification-test/identifiers/identifiers.go

```diff
@@ -382,7 +382,7 @@ func InitCatalog() map[claim.Identifier]claim.TestCaseDescription {
 		DpdkCPUPinningExecProbeRemediation,
 		NoDocumentedProcess,
 		TestDpdkCPUPinningExecProbeDocLink,
-		false,
+		true,
```

Comment:
Shouldn't we mark this implemented only after the tests are tested? technically this is not yet covered by QE.